core/src/main/java/io/lionweb/serialization/data/SerializedClassifierInstance.java

```diff
@@ -169,6 +169,25 @@ public void addChildren(MetaPointer containment, List<String> childrenIds) {
     this.containments.add(new SerializedContainmentValue(containment, childrenIds));
   }
 
+  /**
+   * Removes the specified child from the containments, if present.
+   *
+   * @param child the identifier of the child to be removed; must not be null
+   * @return true if the child was successfully removed, false otherwise
+   */
+  public boolean removeChild(@Nonnull String child) {
```

Comment:
We're talking about the child's **ID** here, right? I'd prefer if the argument‘s name reflected that.